web/themes/new_weather_theme/assets/sass/components/forecast-details-toggle.scss

```diff
@@ -0,0 +1,22 @@
+@use "uswds-core" as *;
+
+.wx-forecast-details-toggle ~ * {
+    display: none;
+}
+
+.wx-forecast-details-toggle ~ h4 {
+    display: block;
+}
+
+.wx-forecast-details-toggle[data-selected="charts"] ~ *:not(wx-hourly-table) {
+    display: initial;
+}
+
+.wx-forecast-details-toggle[data-selected="charts"] ~ *:last-child {
+    display: none;
+}
```

Comment:
So I understand, is this a workaround for the fact that the QPF table and chart are in the same container element? If so, it might be worthwhile to put a comment in here about it for future us's sake.

Reply:
Yes it is, but this is all going to change anyway when we get to https://github.com/weather-gov/weather.gov/issues/1944 because we are going to have to put the different pieces in to different actual tabpanel elements